external-crates/move/crates/move-compiler/src/diagnostics/mod.rs

```diff
@@ -59,7 +59,7 @@ pub struct Diagnostic {
 }
 
 #[derive(PartialEq, Eq, Hash, Clone, Debug, Default)]
-pub struct Diagnostics(Option<Diagnostics_>);
+pub struct Diagnostics(Option<Diagnostics_>, DiagnosticsFormat);
```

Comment:
nit, maybe named fields now?

Reply:
I went back and forth on this, so if you think so, too, I'll do it.